app/src/test/java/org/simple/clinic/patientcontact/PatientContactUiRendererTest.kt

```diff
@@ -0,0 +1,85 @@
+package org.simple.clinic.patientcontact
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.verifyNoMoreInteractions
+import com.nhaarman.mockito_kotlin.verifyZeroInteractions
+import org.junit.Test
+import org.simple.clinic.TestData
+import org.simple.clinic.patient.Age
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.util.TestUserClock
+import org.threeten.bp.Instant
+import org.threeten.bp.LocalDate
+import java.util.UUID
+
+class PatientContactUiRendererTest {
+
+  private val patientUuid = UUID.fromString("e6ff79b9-0ac8-4d7b-ada9-7b3056db2972")
+  private val defaultModel = PatientContactModel.create(patientUuid)
+
+  private val ui = mock<PatientContactUi>()
+  private val clock = TestUserClock(LocalDate.parse("2018-01-01"))
+  private val uiRenderer = PatientContactUiRenderer(ui, clock)
+
+  @Test
+  fun `when the model is not initialized, do nothing`() {
+    // when
+    uiRenderer.render(defaultModel)
+
+    // then
+    verifyZeroInteractions(ui)
```

Comment:
As in? Rename the test method?

Reply:
I just wanted to understand that what are we testing here? What is the expected behavior when the default model is loaded? 
If I am saying do nothing and still testing something being done, it is confusing.
Maybe renaming might help.

Reply:
Ah, this was originally meant to be a regression test to capture accidental calls for a default model. Since the default model actually has actions that are verified in another test, we can remove this.

I will remove this test.